### ISSUE TITLE
Can't access device

### ISSUE THREAD

**Issue by unclegibbo** [hardbyte/ha-evnex#33]
I have just found your integration and am trying to set it up on Home Assistant. I installed HACS and then pulled in this library using the button from the readme, but am wondering if there are more steps involved in setting this up? I would expect I need to enter authentication credentials somewhere but not sure where I would need to add these if I need to do so manually, and am also not receiving any prompt in the UI to do so. I have restarted HA since installing. There is nothing showing up in the logs either. This is what I can see. Any help would be appreciated thanks!
![image](https://github.com/hardbyte/ha-evnex/assets/168252407/2e0971cf-1d2c-4808-8b00-a4b3af356428)

**Comment by hardbyte** [comment]
Yeah there is a section in home assistant called something like developer settings?

**Comment by mikey0000** [comment]
@unclegibbo you need to install it from HACS, then go to devices and services and hit the + button and search for Evnex
